src/events/guildDelete.js

```diff
@@ -0,0 +1,15 @@
+const Event = require("../structures/Event");
```

Comment:
Any specific reason we need a timer instead of just deleting immediately?

Reply:
We certainly don't need to, but it'd be potentially more convenient for the user in case the bot is accidentally or mistakenly kicked, for example.